wagtail/wagtailcore/blocks/stream_block.py

```diff
@@ -168,10 +167,8 @@ def clean(self, value):
                 )
             except ValidationError as e:
                 errors.append(ErrorList([e]))
```

Comment:
``.append()`` needs to be changed to something that works with dicts. (and we're probably missing a test for this branch)